### PR TITLE
Wielding QoL

### PR DESCRIPTION
**Pull request by TheLeopold**
## About The Pull Request
Wielding something will drop item in the offhand
Switching hands while wielding something will unwield it
Trying to wield empty hand will wield offhand item

Added function to drop offhand item

## Why It's Good For The Game
QoL. Makes the combat inventory management more fluid.

## Testing
https://user-images.githubusercontent.com/62264115/234029810-50e9a907-bd71-4631-9c0f-8ac41f83dc36.mp4
Picked and wielded various things via various methods (video)

## Changelog
:cl:
tweak: wielding now is more fluid
code: Added function to drop offhand item
/:cl:

### CODE REVIEW

**Comment by Mycah142** [comment]
Me on my way to accidentally drop my fucking gun because I tried to swap to the OFFHAND icon to hit Q (disabling wielding) so I could tact reload it and fucking die.

**Comment by TheLeopold** [comment]
> Me on my way to accidentally drop my fucking gun because I tried to swap to the OFFHAND icon to hit Q (disabling wielding) so I could tact reload it and fucking die.

Elaborate?

File: code/modules/mob/inventory.dm [hunk 1]
```diff
@@ -79,6 +79,10 @@
 	var/obj/item/I = get_active_hand()
 	unEquip(I, Target, MOVED_DROP)
 
+/mob/proc/drop_offhand(var/atom/Target)
+	var/obj/item/I = get_inactive_hand()
+	unEquip(I, Target, MOVED_DROP)
```
Comment (TheShown911) [inline code/modules/mob/inventory.dm:84]: Would this cause you to drop arm blades or other implants if they are in your offhand when you try to wield?

Comment (TheLeopold) [reply code/modules/mob/inventory.dm:84]: It says it checks if item can be unequipped.
изображение
But I'm booting local to test

Comment (TheLeopold) [reply code/modules/mob/inventory.dm:84]: It doesn't drop the armblade. But says it's dropped. And wields thing with one hand, keeping the armblade.

Comment (TheLeopold) [reply code/modules/mob/inventory.dm:84]: Somewhat fixed it. Armblades and such won't be dropped and won't say they has been dropped. 
And can't be wielded if something in offhand. Will try to address that now.

Comment (TheLeopold) [reply code/modules/mob/inventory.dm:84]: fixed

Comment (Holvr0) [reply code/modules/mob/inventory.dm:84]: Perhaps the deployable cybernetics could auto-retract when you try to wield something in the other hand?

Comment (TheLeopold) [reply code/modules/mob/inventory.dm:84]: They're strong enough without it

**Pull request merged**